README.md

```diff
@@ -1,11 +1,21 @@
-# Gadget
-## The Yeoman Generator for Drupal and Grunt Drupal Tasks
+# Gadget: A Generator for New Drupal Projects
+
+> [Yeoman](http://yeoman.io) generator to scaffold a [Drupal](https://www.drupal.org) project for use with [grunt-drupal-tasks](https://github.com/phase2/grunt-drupal-tasks).
 
-Code Status:
 [![Travis CI status](https://travis-ci.org/phase2/generator-gadget.png?branch=master)](https://travis-ci.org/phase2/generator-gadget)
 [![Dependency Status](https://david-dm.org/phase2/generator-gadget.svg)](https://david-dm.org/phase2/generator-gadget)
 [![npm version](https://badge.fury.io/js/generator-gadget.svg)](https://www.npmjs.com/package/generator-gadget)
 
+## Features
+
+Gadget will set up a Drupal project with the site skeleton and tools to build your project and configure your build system with the Grunt-Drupal-Tasks kit.
```

Comment:
For consistency, let's use "Grunt Drupal Tasks" (as the name) or "grunt-drupal-tasks" (as the package name). The hyphenated version is used more later in this file.